manual/C/ch_Finance-Quote.xml

```diff
@@ -249,6 +249,204 @@ SYNOPSIS
     </procedure>
   </sect1>
 
+  <sect1 id="finance-quote-helper">
+    <title>Helper Scripts</title>
+
+    <abstract>
+      <para>Occasionally, you may need to update the &app-fq; software module separate from &app;.
+        To help with this process, the &app; developers provide several helper programs designed
+        to make it easier to install, update and diagnose problems with your installation of the
+        &app-fq; software module. These programs have names that begin with gnc-fq-*, and are
+        themselves written in the &app-perl; programming language. 
+      </para>
+    </abstract>
+
+    <para>Since these helper programs are &app-perl; scripts, you must have a functioning &app-perl;
+      installation before any of these helper programs will run successfully.
+      (see <xref linkend="finance-quote-perl-install" />)
+    </para>
+
+    <procedure>
+      <title>Running &app-perl; scripts under &win;</title>
+
+      <step>
+        <simpara>Open a command line window. Refer to the manual of your operating
+          system to find out how to do this.
+        </simpara>
+      </step>
+
+      <step>
+        <simpara>Change to the drive containing the &app; program files (usually C drive)
+          if that is not already the current drive.
+        </simpara>
+      </step>
+
+      <step>
+        <simpara>Change to the <userinput>...\gnucash\bin</userinput> directory which contains the
+          required gnc-fq-xxx &app-perl; scripts.
+        </simpara>
+      </step>
+
+      <step>
+        <simpara>Prefix any gnc-fq-xxx &app-perl; script command with <quote>Perl</quote> and a space.
+          <cmdsynopsis>
```

Comment:
cmdsynopsis is intended to build consturcts like in the head of a man page
```
          <cmdsynopsis>
            <command>gnc-fq-dump</command>
            <arg choice="opt">-v</arg>
            <arg choice="req"><replaceable>Source</replaceable></arg>
            <arg choice="req" rep="repeat"><replaceable>Symbol</replaceable></arg>
          </cmdsynopsis>
```
I assume in most cases you want `<screen language="sh">fee fie foo</screen>`
or inline `<userinput>gnc-fq-dump <optional>-v<optional> currency USD EUR<userinput> ` opionally combined with `<computeroutput>1 USD = 0.9778 EUR</computeroutput>`

Reply:
I understand `userinput` in such a way that herewith data, not commands, are marked, which a user has to enter into a program. `Command,` on the other hand, is by definition the markup for the execution of a program.

See [DocBook, userinput](https://tdg.docbook.org/tdg/4.5/userinput.html) and [DocBook, command](https://tdg.docbook.org/tdg/4.5/command.html)

Reply:
As said before while transfering from the wiki replace `<syntaxhighlight language=…>` by `<screen language=…>` or `<programlisting language=…>` and you are done.
Perhaps we should prefer programlisting for pure input and restrict screen to mixed in- and output?

In rare cases it can have a benefit to break a screen down to
```
<computeroutput>prompt</computeroutput><userinput>[settings] command [args]</userinput>
<computeroutput>result</computeroutput>
```
Then there is no language attribute as the interpretation is done by you with your tags.

The main purpose of `<command>` is the usage in cmdsynopsis to describe the formal syntax of a command.

While we are still developing our style, we have before used the command tag different. When we see the old usage, we should replace it, but as it  does not hurt, it is not mandatory.

Perhaps you can add this to the wiki in a way that more people understand?